root@986d73f0cf4d:/workspace/sentence/src# python3 check.py
Using device: cuda
You are using the default legacy behaviour of the <class 'transformers.models.t5.tokenization_t5.T5Tokenizer'>. This is expected, and simply means that the `legacy` (previous) behavior will be used so nothing changes for you. If you want to use the new behaviour, set `legacy=False`. This should only be set if you understand what it means, and thoroughly read the reason why this was added as explained in https://github.com/huggingface/transformers/pull/24565
{'input_ids': tensor([[    2,  2539,  4167,  ...,     0,     0,     0],
        [    2, 24094,  5017,  ...,     0,     0,     0],
        [    2,   591,  1871,  ...,     0,     0,     0],
        ...,
        [    2,  2003,  4793,  ...,     0,     0,     0],
        [    2, 21939, 10400,  ...,     0,     0,     0],
        [    2,  8509,  8295,  ...,     0,     0,     0]], device='cuda:0'), 'attention_mask': tensor([[1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        ...,
        [1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0],
        [1, 1, 1,  ..., 0, 0, 0]], device='cuda:0'), 'labels': tensor([[    2,  2539,  4167,  ...,  -100,  -100,  -100],
        [    2, 24094,  5017,  ...,  -100,  -100,  -100],
        [    2,   591,  1808,  ...,  -100,  -100,  -100],
        ...,
        [    2,  2003,  4793,  ...,  -100,  -100,  -100],
        [    2, 21939, 10400,  ...,  -100,  -100,  -100],
        [    2,  8509,  8295,  ...,  -100,  -100,  -100]], device='cuda:0')}
Traceback (most recent call last):
  File "/workspace/sentence/src/check.py", line 48, in <module>
    assert valid_labels.max().item() < model.dec_tok.vocab_size  # 패딩 제외하고 최대값이 vocab_size보다 작은지 확인
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError
